tests/RegexHelperTest.php

```diff
@@ -37,13 +37,41 @@ public function validVariableValueProvider(): iterable
             'variableValue' => '{{ en-dash }}',
         ];
 
+        yield 'with prefix' => [
+            'variableValue' => '{{ vault.en-dash }}',
+        ];
+
         yield 'no whitespaces' => [
             'variableValue' => '{{nowhitespace}}',
         ];
 
+        yield 'multiple spaces' => [
+            'variableValue' => '{{    whitespace   }}',
+        ];
+
+        yield 'tabs as spaces' => [
+            'variableValue' => "{{\twhitespace\t}}",
+        ];
+
         yield 'everything' => [
             'variableValue' => '{{ Ev3_RY-th1n8 }}',
         ];
+
+        yield 'just numbers' => [
+            'variableValue' => '{{ 123 }}',
+        ];
+
+        yield 'starts with underscore' => [
+            'variableValue' => '{{ _underscore }}',
+        ];
+
+        yield 'just underscore' => [
+            'variableValue' => '{{ _ }}',
+        ];
+
+        yield 'just dash' => [
+            'variableValue' => '{{ - }}',
+        ];
```

Comment:
realne takovy variables jdou zalozit a replacnou se

Reply:
Jasne, ja jsem puvodne vychazal jen z toho omezeni pro Vault, ale to nebylo spravne.